src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +129,13 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sort all persons in list based on alphabetical order of name
+     */
+    public void sort() {
+        internalList.sort(null);
```

Comment:
can restrict implementation to this method by creating custom comparator in Collections.sort()

Reply:
Will consider that!

Reply:
Good point on creating custom comparator and avoid usage of null as much as possible.